words/predicates.txt

```diff
@@ -128,30 +153,40 @@ bubble
 bubbly
 bumpy
 burly
+burnt
 bush
 bustling
 busy
 butter
 buttercup
 buttered
+butternut
+buttery
 button
 buttoned
 cactus
 cake
 calico
 calm
+canary
 candied
 candle
 candy
 canyon
 capable
 capricious
 caramel
+carbonated
 careful
+caribbean
```

Comment:
do we do this with other regions of the world? I can't think of a bad combo off the top of my head so maybe I'm overly worrying, but seems like it could go into offensive land pretty easily?

Reply:
yeah, I wouldn't feel comfortable with (ie) "american" in the list, so lets remove this.